benchmarks/mbpp/mbpp.py

```diff
@@ -0,0 +1,164 @@
+"""
+MBPP: Mostly Basic Python Problems
+
+Jacob Austin, Augustus Odena, Maxwell Nye, Maarten Bosma, Henryk Michalewski,
+David Dohan, Ellen Jiang, Carrie Cai, Michael Terry, Quoc Le, Charles Sutton
+https://arxiv.org/pdf/2108.07732
+
+Based on: https://github.com/google-research/google-research/tree/master/mbpp
+
+# run the eval on the sanitized test split
+inspect eval benchmarks/mbpp/mbpp.py
+"""
+import math  # Used by some assert statements by MBPP.
+
+from datasets import load_dataset
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.model import GenerateConfig
+from inspect_ai.scorer import (CORRECT, INCORRECT, Score, Target, accuracy,
+                               scorer, stderr)
+from inspect_ai.solver import TaskState, generate, prompt_template
+
+PROMPT_TEMPLATE = """
+You are an expert Python programmer. You will be given a task, and the tests that your code must pass.
+Write the Python function to solve the task. Do not give additional explanations, just output the
+Python function. Only use imports that are included in Python's standard library.
+"""
+
+
+def record_to_sample(record):
```

Comment:
Let's move `record_to_sample()` to the very bottom of the source file, and move the `code_acceptance()` scorer right below the task.